### PR TITLE
CoW Protocol: Update Solver Llist

### PR DESCRIPTION
**Pull request by bh2smith**
Adding new production solvers. 

PoC Query: https://dune.com/queries/538257 (working as expected).

### CODE REVIEW

File: ethereum/gnosis_protocol_v2/view_solvers.sql [hunk 1]
```diff
@@ -36,6 +36,18 @@ known_solver_metadata (address, environment, name) as (
                  (decode('a97DEF4fBCBa3b646dd169Bc2eeE40f0f3fE7771', 'hex'), 'prod', 'BalancerSOR'),
                  (decode('6fa201C3Aff9f1e4897Ed14c7326cF27548d9c35', 'hex'), 'prod', 'Otex'),
                  (decode('e58c68679e7aab8ef83bf37e88b18eb1f6e30e22', 'hex'), 'prod', 'PLM'),
```
Comment (nlordell) [inline ethereum/gnosis_protocol_v2/view_solvers.sql:38]: Should we change the table to include a `deprecated` flag and mark the old addresses as such?

Comment (bh2smith) [reply ethereum/gnosis_protocol_v2/view_solvers.sql:38]: We definitely can do this, but that is specifically what the `active` field is already doing.

**Comment by jeff-dude** [comment]
@bh2smith i'm unable to drop due to downstream dependencies on dune user generated table 'dune_user_generated.hour_cow_solver_slippage' -- i'm assuming you created this?

in future PRs, can i just assume it's okay to drop user generated tables and you have the scripts saved to rebuild?

**Comment by bh2smith** [comment]
I did not create that but I am sure someone from my team did (maybe @josojo). In any case, yes absolutely. Please drop cascade here. Whatever is creating that can be rebuilt.

**Review by jeff-dude — APPROVED** [review]
@bh2smith done. will do that without asking next time 👍

**Pull request merged**